pkg/tcpip/transport/tcp/snd.go

```diff
@@ -28,6 +28,9 @@ import (
 )
 
 const (
+	// maxInt is the largest representable value of type int
```

Comment:
Nit: comments be sentences that end with a '.'.

Reply:
Ack.